staging/src/k8s.io/component-base/go.mod

```diff
@@ -20,6 +20,8 @@ require (
 	go.uber.org/atomic v1.4.0 // indirect
 	go.uber.org/multierr v1.1.0 // indirect
 	go.uber.org/zap v1.10.0
+	golang.org/x/oauth2 v0.0.0-20191202225959-858c2ad4c8b6
```

Comment:
Adding dependency here also requires to add it go global `go.mod`. This cause of failure of `pull-kubernetes-dependencies` job. To fix this please run `./hack/update-vendor.sh` or remove those dependencies.